wiki/Tournaments/SOL/2019/en.md

```diff
@@ -1,5 +1,6 @@
 ---
 tags:
+  - SOL
   - SOLMT
 ---
 
```

Comment:
change `It was the first installment of the Speed of Light Mania Tournament.` in line 10 to `It was the first installment of the Speed of Light.`